genai/example_test.go

```diff
@@ -141,6 +143,34 @@ func ExampleGenerativeModel_CountTokens() {
 	fmt.Println("Num tokens:", resp.TotalTokens)
 }
 
+func ExampleGenerativeModel_JSONSchema() {
+	ctx := context.Background()
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer client.Close()
+
+	model := client.GenerativeModel("gemini-1.5-pro")
```

Comment:
Why the model name mismatch with the test? Would `-latest` not work here, or vice versa?